ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/config/index.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { createFileRoute } from '@tanstack/react-router';
+
+import { useRepositoriesServiceGetOrtRunByIndexKey } from '@/api/queries';
+import { RepositoriesService } from '@/api/requests';
+import { LoadingIndicator } from '@/components/loading-indicator';
+import { AdvisorJobDetails } from './-components/advisor-job-details';
+import { AnalyzerJobDetails } from './-components/analyzer-job-details';
+import { EvaluatorJobDetails } from './-components/evaluator-job-details';
+import { NotifierJobDetails } from './-components/notifier-job-details';
+import { ReporterJobDetails } from './-components/reporter-job-details';
+import { ScannerJobDetails } from './-components/scanner-job-details';
+
+const ConfigComponent = () => {
+  const params = Route.useParams();
+
+  const { data: ortRun } = useSuspenseQuery({
+    queryKey: [
+      useRepositoriesServiceGetOrtRunByIndexKey,
+      params.repoId,
+      params.runIndex,
+    ],
+    queryFn: async () =>
+      await RepositoriesService.getOrtRunByIndex({
+        repositoryId: Number.parseInt(params.repoId),
+        ortRunIndex: Number.parseInt(params.runIndex),
+      }),
+  });
+
+  return (
+    <div className='my-2 flex h-full flex-col gap-4 overflow-y-auto'>
```

Comment:
IMO this should not have a scroll bar as seen in the screenshot, but it should just flow down the page and use the browser's scroll functionality.

Reply:
That will have side effects: the sidebar will scroll down with it, becoming invisible, and when reaching the bottom of the long page, also the header will disappear. So this was an easy fix to those effects.

Reply:
I'd just remove `h-full overflow-y-auto` from the div.

Reply:
Well then when clicking a job traffic light, this is the view:
![Capture](https://github.com/user-attachments/assets/a07a75e3-9956-45dd-a848-150e2db2a209)

- sidebar has disappeared
- the top part of the card has disappeared under the header
Probably fixable, but I spent most of yesterday trying. Would appreciate some help, maybe joint debugging?

Reply:
With "sidebar has disappeared", do you mean that it really disappears, or it's just not visible in the viewport as it's higher up on the page?

The top part of the card not being visible can be fixed with this: https://stackoverflow.com/a/69535686

Reply:
I mean just that: it's not visible anymore because it's rolled off the screen. But if this is acceptable, I can finish the PR now.